packages/react-components/react-progress/package.json

```
@@ -32,8 +32,10 @@
     "@fluentui/scripts": "^1.0.0"
   },
   "dependencies": {
+    "@fluentui/react-label": "^9.0.3",
     "@fluentui/react-theme": "^9.0.0",
     "@fluentui/react-utilities": "^9.0.2",
+    "@fluentui/react-shared-contexts": "^9.0.0",
```

nit order alphabetically:

```suggestion
    "@fluentui/react-label": "^9.0.3",
    "@fluentui/react-shared-contexts": "^9.0.0",
    "@fluentui/react-theme": "^9.0.0",
    "@fluentui/react-utilities": "^9.0.2",
```